repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/searchresults-main.ftl

```diff
@@ -39,24 +39,24 @@
             </form>
             <div class="nhsuk-listing">
                 <div class="nhsuk-grid-row">
-                    <div class="nhsuk-grid-column-one-third">
-                        <#-- Filters -->
-                        <@hst.renderURL var="pagelink"/>
-                        <form class="nhsuk-filter" method="get" action="${pagelink}">
-                            <@fmt.message key="filters.label" var="filtersLabel"/>
-                            <p class="nhsuk-filter__title nhsuk-heading-l">${filtersLabel}</p>
+<#--                    <div class="nhsuk-grid-column-one-third">-->
```

Comment:
If it's OK, could we change these inline comments to one block comment so as to avoid FTL engine to process too many comments ?

Reply:
Done

Reply:
Thanks @Raq-HEE - Reverted `ftlvariables` comment updates (as it would be good to keep them as inline comments for easy read) and separated out `contentTypesMap` & `selectedContentTypes` variables for future use if required. Have done this work as part of 29d75f7 commit. Hope this is OK ? Thanks!